spec/models/submission_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Submission, type: :model do
+  describe "validations" do
+    it { should validate_presence_of(:assignment_id) }
```

Comment:
Please check the rspec reviews here and let's sync : https://github.com/nor-wtag/online-marketplace/pull/2